.github/workflows/publish-maven.yml

```diff
@@ -0,0 +1,128 @@
+name: Publish to Maven Central
```

Comment:
This has not yet been tested. We will still release via the release.sh script for now, but this is a start.